core/web/jsonld/PersonTrait.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace luya\web\jsonld;
+
+/**
+ * JsonLd - Person trait
+ *
+ * @see http://schema.org/Person
+ *
+ * @author Alex Schmid
+ * @since 1.0.0
+ */
+trait PersonTrait
+{
+    use ThingTrait {
+        fields as thingFields;
```

Comment:
as field() should not be part of the trait or interface, fields are applied in the object itself this allows us not to rename trait methods, which would lead into not clear design.

Reply:
I thought it would be easier to maintain like that, cause we can avoid the "no single inheritance"-problem...

Reply:
e.g. 
In LocalBusinessTrait we'll merge in PlaceTrait->fields() and OrganisationTrait->fields()

Like this we always have only one place where the code belongs to, so if once schema get's changed or so, there's only one trait we have to do something and not fields everywhere we have to think about...

That was my idea behind it...

Reply:
Ok, its right it could be more easy to maintain the fields. Maybe we could also find getter methods in BaseThing and auto build the fields. This would 

1. fixed the problem with renaming
2. easy to maintain.

As every jsonld element MUST implement BaseThing.